11ty/index.njk

```diff
@@ -15,9 +15,24 @@ layout: layouts/index.njk
       With the foundation of vocabulary, we can begin to understand lived experiences of people different than us. Words can provide us with a sense of identity and allow us to find kinship through common experiences.
     </p>
   </div>
-  <div class="box">
-    <h2 class="sub-headline">Ways to help</h2>
-    <ol class="list">
+  <div
+    class="box spoiler spoiler_collapsed"
+    data-spoiler-mobile-only="true"
+    id="ways-to-help">
+    <h2 class="sub-headline">Ways to help
+      <button
+        aria-expanded="true"
+        aria-controls="ways-to-help-content"
+        class="spoiler__toggle-btn"
+        id="ways-to-help-visibility-toggle">
+        {# https://css.gg/chevron-down #}
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24"><path fill="currentColor" d="M6.34 7.76L4.93 9.17 12 16.24l7.07-7.07-1.41-1.41L12 13.4 6.34 7.76z"/></svg>
+      </button>
+    </h2>
```

Comment:
Blocking: The button has no accessible name at the moment ([see this article for reference](https://www.sarasoueidan.com/blog/accessible-icon-buttons/)). We need to add some to have it make sense for screen reader users.

Reply:
The article was quite helpful, thanks a lot for sharing!

Reply:
Fixed in c73385a